spec/lib/evm_database_spec.rb

```diff
@@ -14,10 +14,31 @@
     end
   end
 
-  context "#seed_primordial" do
-    it "populates seeds" do
-      described_class::PRIMORDIAL_CLASSES.each { |klass| expect(klass.constantize).to receive(:seed) }
-      described_class.seed_primordial
+  context ".seed" do
+    def expect_proper_counts
+      expect(MiqDatabase.count).to eq(1)
+      expect(MiqRegion.count).to eq(1)
+      expect(MiqEnterprise.count).to eq(1)
+      expect(Zone.count).to eq(1)
+      expect(MiqServer.count).to eq(1)
+      expect(ServerRole.count).to eq(23)
+      expect(Tenant.count).to eq(1)
+      expect(MiqProductFeature.count).to eq(950)
```

Comment:
:cry: we probably don't want to do this one.  It's a hassle since we fairly often add product features.

It's already tested elsewhere:  https://github.com/ManageIQ/manageiq/pull/4581